Tools/autotest/param_metadata/rstemit.py

```diff
@@ -224,7 +237,9 @@ def emit(self, g):
 """.format(tag=tag, underline="-" * len(tag),
            reference=reference)
 
-        for param in g.params:
```

Comment:
My undersanding is that we *stopped* sorting parameters because it was causing weird ordering problems in the Wiki.  #22634 is a minimal change just for SIM_

Reply:
this PR only sorts SIM_ parameters